<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogclient</class>
 <widget class="QDialog" name="dialogclient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel {
font-weight: bold;
font-size: 10px;
}
QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}

QLabel {
font-weight: bold;
font-size: 10px;
}
QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}


QPushButton {
color: white;
font-weight: bold;
background-color: black;
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 10px;
padding-left: 5px;
padding-right: 5px;

}
QPushButton:hover { background-color: white;
color:black;
text-decoration: underline; }
</string>
  </property>
  <widget class="QPushButton" name="logout_2">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>10</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/image/Capture53.png);</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/image/Capture53.png);</string>
   </property>
   <property name="text">
    <string>                   CLIENT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logout">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/image/Capture53.png);</string>
   </property>
   <property name="text">
    <string>LOGOUT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="prevcrud">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/image/Capture53.png);</string>
   </property>
   <property name="text">
    <string>←</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextmodule">
   <property name="geometry">
    <rect>
     <x>599</x>
     <y>460</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/image/Capture53.png);</string>
   </property>
   <property name="text">
    <string>→</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>631</width>
     <height>401</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="create">
    <attribute name="title">
     <string>CREATE</string>
    </attribute>
    <widget class="QPushButton" name="ajouter">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>320</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/image/image/Capture53.png);</string>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>91</width>
       <height>251</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>CLIENT ID</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>ID CLIENT :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>DATE</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>NOM</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>PRENOM</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Type of wedding</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>341</width>
       <height>251</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEdit_id">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/image/image/Capture5.png);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_idc">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/image/image/Capture5.png);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_date">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/image/image/Capture5.png);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_nom">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/image/image/Capture5.png);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_prenom">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/image/image/Capture5.png);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Tfete">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/image/image/Capture5.png);</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="read">
    <attribute name="title">
     <string>READ</string>
    </attribute>
    <widget class="QTableView" name="tabcli">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>611</width>
       <height>251</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/image/image/Capture54.png);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="search">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>10</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/image/image/Capture5.png);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="id_asc">
     <property name="geometry">
      <rect>
       <x>-60</x>
       <y>30</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton {
color: grey;
font-weight: bold;
background-color: none;
border-width: none;
border-color: none;
border-style: none;
border-radius: none;
padding: none;


}
QPushButton:hover { background-color: none;
color:black;
text-decoration: none; }</string>
     </property>
     <property name="text">
      <string>                                      ^</string>
     </property>
    </widget>
    <widget class="QPushButton" name="id_desc">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton {
color: grey;
font-weight: none;
background-color: none;
border-width: none;
border-color: none;
border-style: none;
border-radius: none;
padding: none;


}
QPushButton:hover { background-color: none;
color:black;
text-decoration: none; }</string>
     </property>
     <property name="text">
      <string>           v</string>
     </property>
    </widget>
    <widget class="QPushButton" name="num_asc">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton {
color: grey;
font-weight: bold;
background-color: none;
border-width: none;
border-color: none;
border-style: none;
border-radius: none;
padding: none;
font-size: 10px;
padding-left: 5px;
padding-right: 5px;

}
QPushButton:hover { background-color: none;
color:black;
text-decoration: none; }</string>
     </property>
     <property name="text">
      <string>                            ^</string>
     </property>
    </widget>
    <widget class="QPushButton" name="des_asc">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton {
color: grey;
font-weight: bold;
background-color: none;
border-width: none;
border-color: none;
border-style: none;
border-radius: none;
padding: none;
font-size: 10px;
padding-left: 5px;
padding-right: 5px;

}
QPushButton:hover { background-color: none;
color:black;
text-decoration: none; }</string>
     </property>
     <property name="text">
      <string>                            ^</string>
     </property>
    </widget>
    <widget class="QPushButton" name="montant_asc">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>30</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton {
color: grey;
font-weight: bold;
background-color: none;
border-width: none;
border-color: none;
border-style: none;
border-radius: none;
padding: none;
font-size: 10px;
padding-left: 5px;
padding-right: 5px;

}
QPushButton:hover { background-color: none;
color:black;
text-decoration: none; }</string>
     </property>
     <property name="text">
      <string>                            ^</string>
     </property>
    </widget>
    <widget class="QPushButton" name="type_asc">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>30</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton {
color: grey;
font-weight: bold;
background-color: none;
border-width: none;
border-color: none;
border-style: none;
border-radius: none;
padding: none;
font-size: 10px;
padding-left: 5px;
padding-right: 5px;

}
QPushButton:hover { background-color: none;
color:black;
text-decoration: none; }</string>
     </property>
     <property name="text">
      <string>     ^</string>
     </property>
    </widget>
    <widget class="QPushButton" name="type_desc">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>30</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton {
color: grey;
font-weight: none;
background-color: none;
border-width: none;
border-color: none;
border-style: none;
border-radius: none;
padding: none;


}
QPushButton:hover { background-color: none;
color:black;
text-decoration: none; }</string>
     </property>
     <property name="text">
      <string>  v</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/image/image/Capture53.png);</string>
     </property>
     <property name="text">
      <string>PRINT</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>350</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/image/image/Capture53.png);</string>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>350</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/image/image/Capture53.png);</string>
     </property>
     <property name="text">
      <string>Modify</string>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>591</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="txt_id">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/image/image/Capture5.png);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_num">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/image/image/Capture5.png);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_des">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/image/image/Capture5.png);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_mont">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/image/image/Capture5.png);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_type">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/image/image/Capture5.png);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_tfete">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/image/image/Capture5.png);</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="type_asc_2">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>30</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton {
color: grey;
font-weight: bold;
background-color: none;
border-width: none;
border-color: none;
border-style: none;
border-radius: none;
padding: none;
font-size: 10px;
padding-left: 5px;
padding-right: 5px;

}
QPushButton:hover { background-color: none;
color:black;
text-decoration: none; }</string>
     </property>
     <property name="text">
      <string>              ^</string>
     </property>
    </widget>
    <widget class="QPushButton" name="type_desc_2">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>30</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton {
color: grey;
font-weight: none;
background-color: none;
border-width: none;
border-color: none;
border-style: none;
border-radius: none;
padding: none;


}
QPushButton:hover { background-color: none;
color:black;
text-decoration: none; }</string>
     </property>
     <property name="text">
      <string>    v</string>
     </property>
    </widget>
    <widget class="QPushButton" name="num_desc">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton {
color: grey;
font-weight: bold;
background-color: none;
border-width: none;
border-color: none;
border-style: none;
border-radius: none;
padding: none;
font-size: 10px;
padding-left: 5px;
padding-right: 5px;

}
QPushButton:hover { background-color: none;
color:black;
text-decoration: none; }</string>
     </property>
     <property name="text">
      <string>                            v</string>
     </property>
    </widget>
    <widget class="QPushButton" name="des_desc">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton {
color: grey;
font-weight: bold;
background-color: none;
border-width: none;
border-color: none;
border-style: none;
border-radius: none;
padding: none;
font-size: 10px;
padding-left: 5px;
padding-right: 5px;

}
QPushButton:hover { background-color: none;
color:black;
text-decoration: none; }</string>
     </property>
     <property name="text">
      <string>                            v</string>
     </property>
    </widget>
    <widget class="QPushButton" name="montant_desc">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>30</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton {
color: grey;
font-weight: bold;
background-color: none;
border-width: none;
border-color: none;
border-style: none;
border-radius: none;
padding: none;
font-size: 10px;
padding-left: 5px;
padding-right: 5px;

}
QPushButton:hover { background-color: none;
color:black;
text-decoration: none; }</string>
     </property>
     <property name="text">
      <string>                            v</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>10</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/image/image/Capture53.png);</string>
     </property>
     <property name="text">
      <string>STATISTIQUE</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="update">
    <attribute name="title">
     <string>UPDATE</string>
    </attribute>
    <widget class="QLineEdit" name="idEdit">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>70</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
border-image: url(:/image/image/Capture5.png);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="edit">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>70</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/image/image/Capture53.png);</string>
     </property>
     <property name="text">
      <string>Select</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>ID :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_idc">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>140</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
border-image: url(:/image/image/Capture5.png);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editDate">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>140</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
border-image: url(:/image/image/Capture5.png);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_nom">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>230</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
border-image: url(:/image/image/Capture5.png);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_prenom">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>230</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
border-image: url(:/image/image/Capture5.png);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="editOk">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>340</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/image/image/Capture53.png);</string>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>NEW ID CLIENT : </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>130</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>NEW DATE :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>230</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>NEW NOM :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>230</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>NEW PRENOM :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>295</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>NEW TYPE DE FETE :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_tfete">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>299</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/image/image/Capture5.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="deletee">
    <attribute name="title">
     <string>DELETE</string>
    </attribute>
    <widget class="QPushButton" name="delete_btn">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>100</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/image/image/Capture53.png);</string>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>47</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>ID :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="deleteLine">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>291</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/image/image/Capture5.png);</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="previousmodule">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>460</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/image/Capture53.png);</string>
   </property>
   <property name="text">
    <string>←</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>460</y>
     <width>551</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/image/Capture53.png);</string>
   </property>
   <property name="text">
    <string>MAINMENU</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextcrud">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/image/Capture53.png);</string>
   </property>
   <property name="text">
    <string>→</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
